packages/auth0/README.md

```diff
@@ -71,58 +74,63 @@ Use the values returned from the query to update your configuration in the clien
 }
 ```
 
-Create a fake user whose credentials can be used for authentication with this query:
+Create a fake user whose credentials can be used for authentication with this query.
 
 ```graphql
-mutation{
-  given(a: "person", simulation:"6fbe024f-2316-4265-a6e8-d65a837e308a")
+mutation {
+  given(a: "person", simulation: "6fbe024f-2316-4265-a6e8-d65a837e308a")
 }
 ```
 
 ![person query](./docs/person.png)
 
-Use the `email` and `password` fields as login credentials.
+The fake user can now be used on your app. Use the `email` and `password` fields as login credentials.
 
 You now have a running auth0 server!
 
-### code
+### Code
+
+An auth0 simulator can be created using the `@simulacrum/client` package. This is how you would apply the mutations programmatically.
 
-An auth0 simulator can be created using the `@simulacrum/client` package:
+The following examples are written in Typescript, but using Typescript is not a requirement. The Auth0 simulator creates a server with a graphql interface. This means that your interactions with the server can be written in any language or framework that can communicate over http / graphql.
 
 ```ts
-import { main } from 'effection';
-import { createSimulationServer, Server } from '@simulacrum/server';
-import { auth0 } from '.';
-import { createClient } from '@simulacrum/client';
+import { main } from "effection";
+import { createSimulationServer, Server } from "@simulacrum/server";
+import { auth0 } from ".";
+import { createClient } from "@simulacrum/client";
 
 const port = Number(process.env.PORT) ?? 4400;
 
-main(function*() {
+// effection is a structured concurrency library and
+// this will help us handle errors and shutting down
+// the server gracefully
+main(function* () {
   let server: Server = yield createSimulationServer({
     seed: 1,
     port,
-    simulators: { auth0 }
+    simulators: { auth0 },
   });
 
   let url = `http://localhost:${server.address.port}`;
 
   console.log(`simulation server running at ${url}`);
 
-  let client = yield createClient(url);
```

Comment:
@dagda1 I removed the `yield` here. It was throwing a loop error in effection. Just noting in case this shows up somewhere else that I didn't notice.

Reply:
`createClient` is indeed a synchronous function. Also, might be worth showing usage from `async/await` since client is designed to hide the fact that it uses effection internally.

Reply:
Is `createSimulationServer` designed that way as well? Do we still need to wrap it with effection `main` if we are using `async`/`await`? I can't seem to get the server to start using `async`/`await`.

Reply:
@jbolda No, `createSimulationServer` must be used as an Effection operation. The client on the other hand, will be running inside all kinds of enviroments; mostly cypress, jest, mocha test suites, etc... and so for the client only we opted for making it just simple `async/await` and not assume an effection environment.

The simplest integration is to use `main()` or `run()` if you need to and know what you're doing.

Reply:
@cowboyd I added an additional example that includes async/await. Let me know what you think.